### PR TITLE
Fixes various 404 page issues

### PR DESCRIPTION
**Pull request by pjaclark**
Working through 404 pages from Google Search Console, fixing issues.

### CODE REVIEW

File: content/riak/ts/1.3.1/installing/aws.md [hunk 1]
```diff
@@ -12,8 +12,7 @@ project_version: "1.3.1"
 toc: true
 version_history:
   locations:
-    - ["1.3.0-1.3.1", "installing/aws"]
-    - ["1.4.0+",      "setup/installing/aws"]
+  in: "1.3.1+"
```
Comment (pjaclark) [inline content/riak/ts/1.3.1/installing/aws.md:15]: Why remove the 1.3.0? And why remove the locations for the 1.4.0+

File: content/riak/ts/1.3.1/installing/source/erlang.md [hunk 2]
```diff
@@ -11,9 +11,8 @@ project: "riak_ts"
 project_version: "1.3.1"
 toc: true
 version_history:
-  locations:
-    - ["1.3.0-1.3.1", "installing/source/erlang"]
-    - ["1.4.0+",      "setup/installing/source/erlang"]
+version_history:
+  in: "1.3.1+"
```
Comment (pjaclark) [inline content/riak/ts/1.3.1/installing/source/erlang.md:15]: As above

File: content/riak/ts/1.4.0/using/querying/select/arithmetic-operations.md [hunk 3]
```diff
@@ -11,9 +11,7 @@ project: "riak_ts"
 project_version: "1.4.0"
 toc: true
 version_history:
-  present_from: "1.4.0+"
-  moved:
-    - ["1.1.0+", "using/arithmetic-operations"]
+  in: "1.4.0+"
```
Comment (pjaclark) [inline content/riak/ts/1.4.0/using/querying/select/arithmetic-operations.md:14]: Should also have a `locations` with 1.1.0-1.3.1 being `using/arithmetic-operations/` and 1.4.0 being `using/querying/select/arithmetic-operations/`?

File: content/riak/ts/1.4.0/using/riakshell.md [hunk 4]
```diff
@@ -10,6 +10,8 @@ menu:
 project: "riak_ts"
 project_version: "1.4.0"
 toc: true
+version_history:
+  in: "1.4.0+"
```
Comment (pjaclark) [inline content/riak/ts/1.4.0/using/riakshell.md:14]: Goes back to 1.2.0

File: content/riak/ts/1.3.1/installing/aws.md (hunk 1 above)
Comment (JMercerGit) [reply content/riak/ts/1.3.1/installing/aws.md:15]: Might have made a mistake here. I'll revert this one and the one below

**Comment by pjaclark** [comment]
LGTM

**Comment by pjaclark** [comment]
Same for `content/riak/ts/1.4.0/using/riakshell.md`

**Comment by pjaclark** [comment]
And `content/riak/ts/1.3.1/installing/source/erlang.md`.

At this point, I would suggest checking all the changes to be consistent across versions.

**Pull request merged**